internal/cmd/kafka/command_cluster_update.go

```diff
@@ -34,6 +36,8 @@ func (c *clusterCommand) newUpdateCommand(cfg *v1.Config) *cobra.Command {
 	}
 
 	cmd.Flags().String("name", "", "Name of the Kafka cluster.")
+	pcmd.AddAvailabilityFlag(cmd)
+	pcmd.AddTypeFlag(cmd)
```

Comment:
What is the difference between lines 38/41 and 39/40? It seems like we're missing a description or information in the case the user requests help.